src/main/resources/assets/forestry/lang/cs_CZ.lang

```diff
@@ -0,0 +1,914 @@
+####################
+# CORE
+####################
+
+yes=Yes
```

Comment:
It looks like these keys are extremely outdated and will not work at all.
Please do a quick look and remove any lang files that have never been updated since like 1.6.4, they just add size to the jar. Anyone who wants to translate them in the future can look at the old repo as a starting point.